src/components/Board/Board.component.js

```diff
@@ -104,7 +107,71 @@ export class Board extends Component {
 
     this.state = {
       openTitleDialog: false,
-      titleDialogValue: props.board && props.board.name ? props.board.name : ''
+      titleDialogValue: props.board && props.board.name ? props.board.name : '',
+      runLiveHelp: false,
+      runLiveUnlock: false,
+      hideCloseButton: true,
+      steps: [
+        {
+          target: 'body',
+          placement: 'center',
+          hideCloseButton: true,
+          content: (
+            <h2>
+              <FormattedMessage {...messages.walkthroughStart} />
+            </h2>
+          )
+        },
+        {
+          hideCloseButton: true,
+          target: '.personal__account',
+          content: <FormattedMessage {...messages.walkthroughSignInUp} />
+        },
+        {
+          hideCloseButton: true,
+          target: '.edit__board__ride',
+          content: <FormattedMessage {...messages.walkthroughEditBoard} />
+        },
+        {
+          hideCloseButton: true,
+          target: '.EditToolbar__BoardTitle',
+          content: <FormattedMessage {...messages.walkthroughBoardName} />
+        },
+        {
+          hideCloseButton: true,
+          target: '.add__board__tile',
+          content: <FormattedMessage {...messages.walkthroughAddTile} />
+        },
+        {
+          hideCloseButton: true,
+          target: '.Communicator__title',
+          content: <FormattedMessage {...messages.walkthroughChangeBoard} />
+        },
+        {
+          hideCloseButton: true,
+          target: '.edit__communicator',
+          content: (
+            <FormattedMessage {...messages.walkthroughBuildCommunicator} />
+          )
+        }
+      ],
+      firststep: [
```

Comment:
We want to remove `firststep` from the `state` object and put in as a class property (for performance reasons).